### PR TITLE
add High-Mobility

### PR DESCRIPTION
**Pull request by slashmili**
Please review our [Contribution Guidelines](https://github.com/doomspork/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.

Review your changes and complete the checklist below.

- [ ] Company name
- [ ] Company description
- [ ] Company location
- [ ] GitHub link
- [ ] How Elixir is used
- [ ] Sorted alphabetically

### CODE REVIEW

**Comment by slashmili** [comment]
not sure what's wrong with `fredapp.com.br` maybe @alfredbaudisch can help?

File: .travis.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ rvm:
 before_script:
   - gem install awesome_bot
 script:
-  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com
+  - awesome_bot README.md --allow-redirect -w agilion.com,brighterlink.io,dubber.net,therealreal.com,marketfactory.com,www.high-mobility.com
```
Comment (doomspork) [inline .travis.yml]: I would rather you not add yourself to the whitelist, this is more for exceptions where the awesome_bot doesn't handle redirects correctly.

**Comment by doomspork** [comment]
I'm not too worried about failing builds, we use that to keep an eye on companies that go out of business and need to be removed.

File: .travis.yml (hunk 1 above)
Comment (slashmili) [inline .travis.yml]: @doomspork If I don't, the build fails with [this error](https://travis-ci.org/doomspork/elixir-companies/builds/268038755#L586):

```
 1. [L008]  https://www.high-mobility.com/ SSL_connect SYSCALL returned=5 errno=0 state=unknown state 
```

**Comment by slashmili** [comment]
@doomspork I removed our domain from whitelist

**Comment by doomspork** [comment]
Thank you @slashmili! 😁

**Pull request merged**